hack/test-integration.sh

```diff
@@ -86,6 +86,7 @@ function exectest() {
 
 	if [[ ${result} -eq 0 ]]; then
 		os::text::print_green "ok      $1"
+		rm -rf /tmp/etcd?????????
```

Comment:
@liggitt will you be ok if we remove the directory in the test itself? dunno if gotest allows something like 'cleanup'. If not we can add it.